Data/Primitive/PrimRef.hs

```diff
@@ -0,0 +1,146 @@
+{-# LANGUAGE CPP #-}
+{-# LANGUAGE MagicHash #-}
+{-# LANGUAGE RoleAnnotations #-}
+{-# LANGUAGE UnboxedTuples #-}
+{-# LANGUAGE Unsafe #-}
+
+-- | Note: Edward Kmett wrote everything in this module. 
```

Comment:
```suggestion
-- | Unboxed mutable variables.
--
-- Note: Edward Kmett wrote everything in this module. 
```
Should we keep the note about Kmett here? The code is pretty trivial, it's just wrapping `MutablePrimArray` functions and some atomic functions from `GHC.Exts`. The original also used `MutableByteArray`.

Reply:
I've rewritten this header in a PR against this branch. I think it's good to mention the original author.